ozaria/site/components/play/PageUnitMap/common/UnitMapLevelDot.vue

```diff
@@ -76,7 +76,7 @@
       this.setLevelConcepts()
     },
     mounted () {
-      $('.has-tooltip').tooltip({ html: true })
+      $('.level-dot-image.has-tooltip').tooltip({ html: true })
```

Comment:
This change is to make the Unit map tooltips more specific to only the unit map. Otherwise additional tooltips are applied to the chrome menu.